server/lib/src/idm/account.rs

```diff
@@ -69,6 +70,7 @@ pub struct Account {
     pub(crate) sshkeys: BTreeMap<String, SshPublicKey>,
     pub apps_pwds: BTreeMap<Uuid, Vec<ApplicationPassword>>,
     pub(crate) oauth2_client_provider: Option<OAuth2AccountCredential>,
+    pub updated_at: Option<u64>,
```

Comment:
This should probably be an `OffsetDateTime` instead (tbf, as a user, I'd prefer it to be a strongly typed value everywhere)

Reply:
Ah, right, sorry, confused with SCIM

Reply:
In this step we should be pulling the LastModifiedCID and leaving it as the CID until the *last possible moment* when we yield it to the client. We can only "narrow" our information, so the sooner you are removing the CID, the less info we have LATER in the process potentially. 

Think of it as "data as the server knows" vs "data as presented". You're setting it up for presentation *too soon*.